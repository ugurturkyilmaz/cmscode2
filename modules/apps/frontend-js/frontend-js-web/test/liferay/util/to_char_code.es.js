/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

import toCharCode from '../../../src/main/resources/META-INF/resources/liferay/util/to_char_code.es';

describe('Liferay.Util.toCharCode', () => {
	it('returns string', () => {
		const result = toCharCode('liferay');

		expect(typeof result).toBe('string');
		expect(result).toMatchSnapshot();
	});
});
