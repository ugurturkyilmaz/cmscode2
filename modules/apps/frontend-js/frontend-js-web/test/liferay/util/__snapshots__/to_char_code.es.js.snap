// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Liferay.Util.toCharCode returns string 1`] = `"10810510210111497121"`;
